docker/www/Dockerfile

```diff
@@ -26,6 +26,8 @@ RUN curl -o /usr/local/bin/composer https://getcomposer.org/composer.phar && chm
 
 RUN echo "sendmail_path=/usr/local/bin/mhsendmail" >> /usr/local/etc/php/conf.d/php.ini
 
+RUN curl -fsSL https://deb.nodesource.com/setup_11.x | bash - && apt-get install -y nodejs
```

Comment:
Nie lepiej użyć opcji [`FROM`](https://docs.docker.com/engine/reference/builder/#from) z podaniem [wersji 11.9.0 z Docker Huba](https://hub.docker.com/layers/node/library/node/11.9.0-alpine/images/sha256-30afdc3023f2c05aacca41cadd49a46203930861f1279ca9eae314b36d5d9c72?context=explore)?

Reply:
Teoretycznie można użyć wielu FROM w jednym Dockerfile, ale to chyba lekko komplikuje całość, nie wiem czy nam potrzeba - https://docs.docker.com/develop/develop-images/multistage-build/ A jako osobny kontener w tym przypadku nie za bardzo jest co postawić, bo my tam nie mamy czego uruchomić na stałe, jako entrypoint. Tu trzeba po prostu czasem dostępu do node i npma, aby uruchomić jakiś rebuild.